mmv1/templates/terraform/examples/managedkafka_topic_basic.tf.erb

```diff
@@ -0,0 +1,34 @@
+resource "google_managed_kafka_cluster" "<%= ctx[:primary_resource_id] %>" {
+  cluster_id = "<%= ctx[:vars]['cluster_id'] %>"
+  location = "us-central1"
+  capacity_config {
+    vcpu_count = 3
+    memory_bytes = 3221225472
+  }
+  gcp_config {
+    access_config {
+      network_configs {
+        subnet = "projects/${data.google_project.project.number}/regions/us-central1/subnetworks/default"
+      }
+    }
+  }
+
+  provider = google-beta
+}
+
+resource "google_managed_kafka_topic" "<%= ctx[:primary_resource_id] %>" {
+  topic_id = "<%= ctx[:vars]['topic_id'] %>"
+  cluster = google_managed_kafka_cluster.<%= ctx[:primary_resource_id] %>.cluster_id
```

Comment:
Also hard-code this as a string to reference the cluster rather than <%= ctx[:primary_resource_id] %>